biz.aQute.junit/src/aQute/junit/ConsoleReporter.java

```diff
@@ -0,0 +1,53 @@
+package aQute.junit;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import aQute.lib.exceptions.Exceptions;
+import junit.framework.AssertionFailedError;
+import junit.framework.Test;
+import junit.framework.TestListener;
+
+public class ConsoleReporter implements TestListener {
```

Comment:
No, you cannot get rid of the Tee streams. They are necessary to capture any output of the test cases. (In fact I have a small PR to make to fix a bug I found there while reviewing this PR.)